.github/workflows/areyouok-links.yml

```diff
@@ -0,0 +1,25 @@
+name: Audit Links using AreYouOk
+on:
+  workflow_dispatch:
+  schedule:
+    # At 06:00 AM, every 30 days
+    - cron:  '0 6 */30 * *'
+      
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download & Run AreYouOk
+        run: |
+          wget -q https://github.com/Bhupesh-V/areyouok/releases/latest/download/areyouok-linux-amd64
+          chmod +x areyouok-linux-amd64
+          ./areyouok-linux-amd64 -i=_layouts,.git,_site,build.py,USAGE.md -r=github
+      - name: Create Issue
+        uses: peter-evans/create-issue-from-file@v2
+        with:
+          title: AreYouOk URL Health Report
+          content-filepath: ./report.github
+      - name: Cleanup
```

Comment:
Not sure if the following step is necessary at all since each workflow triggered is ephemeral. So, no reports & generated files will be stored which will be shared across other workflows as well.

But regardless @Bhupesh-V can you confirm this? I might be wrong here.

Reply:
Agreed, Yeah we can let this go, i added this in blog just for sanity, we can remove this Cleanup step